draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1559,6 +1559,8 @@ With AEGIS-256, random nonces can be used with no practical limits.
 
 Regardless of the variant, the `key` and `nonce` are only required by the `Init` function; other functions only depend on the resulting state. Therefore, implementations can overwrite ephemeral keys with zeros right after the last `Update` call of the initialization function.
 
+Both algorithms can be used as a MAC by performing encryption with the message as the `ad` and leaving `msg` empty, resulting in just a tag. However, they MUST NOT be used as a hash function; if the key is known, inputs generating state collisions can easily be crafted. Similarly, as opposed to hash-based MACs, tags MUST NOT be used for key derivation as there is no proof they are uniformly random.
```

Comment:
"performing encryption with the message" can be a bit confusing, as we don't encrypt anything. Maybe replace it with "by calling the `Encrypt()` function"?

Reply:
"Both algorithms" -> maybe "All the variants"?

Reply:
Both good suggestions. I put `Both algorithms` because that was used for the nonce-respecting setting part, which we might want to change.

Reply:
Now I'm debating `Each variant`, but then `However, they` has to change.